app/core/models.py

```diff
@@ -426,3 +426,11 @@ class SocMajor(AbstractBaseModel):
 
     def __str__(self):
         return self.title
+
+
+class UserStatusType(AbstractBaseModel):
```

Comment:
Now that this model is going to be implemented, we should uncomment the one-to-many relationship [here](https://github.com/hackforla/peopledepot/blob/53b4a22507d040a9af5cc45a378b48afb01d8e79/app/core/models.py#L77), according to the tool tip under [here](https://github.com/hackforla/peopledepot/blob/53b4a22507d040a9af5cc45a378b48afb01d8e79/app/core/models.py#L77).